core/domain/stats_services.py

```diff
@@ -40,50 +40,85 @@ def get_top_unresolved_answers_for_default_rule(exploration_id, state_name):
     }
 
 
-def get_exps_unresolved_answers_count_for_default_rule(exp_ids):
-    """Gets answer counts per exploration for the answer groups for default
-    rule across all states for explorations with ids in exp_ids.
+def get_exps_unresolved_answers_for_default_rule(exp_ids):
+    """Gets unresolved answers per exploration for default rule across all
+    states for explorations with ids in exp_ids.
 
-    Note that this method currently returns the counts only for the DEFAULT
-    rule. This should ideally handle all types of unresolved answers.
+    Note that this method currently returns the data only for the DEFAULT rule.
+    This should ideally handle all types of unresolved answers.
 
-    Returns:
-        A dict, keyed by the string '{exp_id}', whose values are the number of
-        unresolved answers that exploration has. Any exp_ids for explorations
-        that don't exist or that have been deleted will be ignored, and not
-        included in the return value.
+    Returns a dict of the following format:
+        {
+          'exp_id_1': {
+            'count': 10 (number of unresolved answers for this exploration),
+            'unresolved_answers': (list of unresolved answers sorted by count)
+              [
+                {'count': 4, 'value': 'answer_1', 'state': 'Introduction'},
```

Comment:
^Scratch that. Sorry, the individual counts should add up to the total count and the whole list is represented here. Edited the docstring to make this clear!